lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -24,7 +24,9 @@ use OpenQA::Schema::Result::Jobs;
 sub _group_result {
     my ($self, $group, $limit) = @_;
 
-    my $timecond = {">" => time2str('%Y-%m-%d %H:%M:%S', time - 24 * 3600 * 14, 'UTC')};
+    my $time_limit_days = $self->param('time_limit_days') // 14;
```

Comment:
Agree, I was only curious. Sometimes when script dies in Mojo controller it doesn't run that code anymore until restarted. I was worried being it this problem, but looks like it survived.